web/docs/Backend Starters/Decoupled WordPress/managing-dependencies.md

```diff
@@ -0,0 +1,30 @@
+# Managing WordPress Dependencies
+
+## 1. Adding a new plugin/theme:
+
+This WordPress Project uses [composer](https://getcomposer.org) to manage dependencies. A new plugin/theme can be added using the following composer require command:
+
+```
+composer require wpackagist-plugin/akismet
+```
+
+For more info:
+
+- https://docs.roots.io/bedrock/master/composer/
+
+## 2. Update dependencies manually using composer
+
+The dependencies can be updated using the following composer update command:
```

Comment:
Same issue here, let's have a generic example first, followed by the akismet example.